data/json/recipes/other/parts_construction.json

```diff
@@ -221,6 +221,22 @@
     "type": "recipe",
     "activity_level": "MODERATE_EXERCISE",
     "result": "wooden_post_long",
+    "id_suffix": "from wood beam by hand",
```

Comment:
While technically allowed, please use single words in accordance with all the other recipes in the game. i.e. this would be `from_wood_beam_by_hand`.

Same for the other one.

Reply:
I changed them.